challenge-04/README.md

```diff
@@ -75,7 +110,41 @@ mostrar quantos assentos ainda podem ser ocupados, com a frase:
 - Se couber somente mais uma pessoa, mostrar a palavra "pessoa" no retorno
 citado acima, no lugar de "pessoas".
 */
-?
+
+
+carro.adicionarPessoas = function(pessoasEntrando) {
+    let assentosDisponiveis = this.assentos - this.quantidadePessoas
+    let validarQuantidadePessoas = 'pessoas'
+   
+    if (pessoasEntrando >= 1) {
+
+        if (this.quantidadePessoas == this.assentos) {
+            return "O carro já está lotado!";
+        } else if (pessoasEntrando > assentosDisponiveis) {
+            if (assentosDisponiveis == 1) {
+                validarQuantidadePessoas = 'pessoa';
+            }
+            return "Só cabem mais " + assentosDisponiveis + " " + validarQuantidadePessoas + " no carro.";
+        } else {
+            this.quantidadePessoas += pessoasEntrando;
+            if (this.quantidadePessoas == 1) {
+                validarQuantidadePessoas = 'pessoa';
+            }
+            return "Já temos " + this.quantidadePessoas + " " + validarQuantidadePessoas + " no carro!";
+        }
+    } else {
+        return "Número de pessoas inválido."
+    }
+}
+
+carro.removerPessoas = function(pessoasSaindo) {
+    if (this.quantidadePessoas >= 1) {
```

Comment:
Validar o valor de pessoasSaindo. Se o valor for maior que zero, você vai ter um valor negativo de pessoas